include/mamba/api/install.hpp

```diff
@@ -7,36 +7,10 @@
 #ifndef MAMBA_API_INSTALL_HPP
 #define MAMBA_API_INSTALL_HPP
 
-#include "mamba/core/context.hpp"
-#include "mamba/core/mamba_fs.hpp"
-#include "mamba/core/package_info.hpp"
-#include "mamba/core/pool.hpp"
-#include "mamba/core/repo.hpp"
-#include "mamba/core/solver.hpp"
-
-#include <string>
-#include <vector>
-
 
 namespace mamba
 {
     void install();
-
-    void install_specs(const std::vector<std::string>& specs,
```

Comment:
hmm, I find moving these to `create.hpp` a bit counterinutive ... 

two other options: 

- merging create and install?
- adding a third file with these implementations?

Reply:
Yeah maybe! I did that to avoid circular import. And it looked quite natural that create is a superset of install.
Let me find out something :)